code/switch.js

```diff
@@ -0,0 +1,332 @@
+      `Which salad do you prefer, ${customerName}?
+
+      1 - Chicken
+      2 - Vegetarian`
+    );
+
+    console.log("Subtype input: ", subType);  // Log to console
+
+    // Convert menu number to text
+    switch (subType) {
+      case "1":
+        subType = "with chicken";
+        break;
+
+      case "2":
+        subType = "without chicken";
+        break;
+
+        case null:
+          exit(1);
+
+      default:
+        subType = "without chicken";
+        alert(
+          `That's not a choice. You'll get a vegetarian 😘`
+        );
+        break;
+    };
+    break;
+};
+
+console.log("Sub type: ", subType);  // Log to console
+
+// 3.b Alert subtype choice
+alert(`Awesome, ${subType} it is.`);
+
+// Step 4 - Age
+// Your code goes here
+
+// 4.a Prompt for age
+const age = prompt(
+  `Depending on your age, we'll make you a child or adult size portion. (No, you can't descide on your own).
+
+  How old are you, ${customerName}?`
+);
+
+console.log("Age input: ", age);  // Log to console
+
+// Descide child or adult size portion
+const isChild = (age < 18); // Is the customer under 18 yo?
+const portionSize = isChild ? "A child's" : "An adult's"; // Set portion size to dispay in order message
+
+console.log("Portion size: ", portionSize);  // Log to console
+
+// Set price
+let price;
+
+switch (foodChoice) {
+  case "pizza":
+    price = isChild ? 90 : 125;
+    break;
+
+  case "pasta":
+    price = isChild ? 100 : 135;
+    break;
+
+  case "salad":
+    price = isChild ? 70 : 100;
+    break;
+};
+
+console.log("Price: ", price); // Log to console
+
+// 4.b Alert price
+alert(`${portionSize} portion is ${price} SEK.`);
+
+// 4.c Prompt price confirmation
+let priceConfirmation = prompt(
+  `Is that OK?
+
+  - YES
+  - NO`
+);
+console.log("Price confirmation: ", priceConfirmation); // Log to console
+
+// Act on price confirmation
+switch (priceConfirmation) {
+  case "YES": // if YES, continue to order confirmation
+    break;
+
+  default: // if NO or nothing, ask if order should be canceled
+    priceConfirmation = prompt(
+      `Ok, do you want to cancel your order?
+    
+      YES - Cancel my order
+      NO - Price is OK`
+    );
+
+    console.log("Cancel order? ", priceConfirmation); // Log to console
+
+    // if "YES" cancel order, otherwise set to confirmed
+    switch (priceConfirmation) {
+      case "YES":
+        priceConfirmation = "NO";
+        alert("Ok, your order is cancelled. Bye!")
+        console.log("Order cancelled"); // Log to console
+        exit(1);
+
+        default:
+          priceConfirmation = "YES";
+          console.log("Continue to order confirmation");
+          break;
+    };  
+    break;
+};
+
+// Step 5 - Order confirmation
+// Your code goes here
+
+// Prompt customer to confirm order
+const orderConfirmation = prompt(
+  `Ok, so you've ordered ${portionSize.toLowerCase()} portion of ${foodChoice} ${subType} for ${price} SEK. Sounds good?
+
+  - YES
+  - NO`
+);
```

Comment:
this step is a bit redundant, could as well skip to the last confirmation message 😄